prog/weave-kube/weave-daemonset-k8s-1.6.yaml

```diff
@@ -47,6 +47,44 @@ items:
       - kind: ServiceAccount
         name: weave-net
         namespace: kube-system
+  - apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
You probably want to update `prog/weave-kube/weave-daemonset-k8s-1.7.yaml` as well. Also, LG (Launch Generator).